### ISSUE TITLE
Missing labels for training scheme columns

### ISSUE THREAD

**Issue by mzur** [biigle/maia#86]
In the new MAIA ob form, the training scheme definition has no labels for the columns. The labels should be "layers", "epochs", "learning rate".

**Comment by mzur** [comment]
In the new new MAIA job form, there is no training scheme configuration any more.
